src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -238,6 +240,15 @@ public CompletableFuture<String> lookupAcquisitionUnitDefault(String data) {
 
   }
 
+  public CompletableFuture<Object> lookupAcquisitionUnitIdsByAccount(String data) {
+
+    return lookupOrganization(ORGANIZATION_NAME).thenApply(org -> org.getAccounts().stream()
+      .filter(acc -> HelperUtils.normalizeSubAccout(acc.getAccountNo()).equals(HelperUtils.normalizeSubAccout(data)))
```

Comment:
nitpick: `HelperUtils.normalizeSubAccount(data)` should be the same value for execution of this lambda. Consider saving its output into a variable and using that instead.

Reply:
actually no need to normalize values of stored organization. Only gobi request can contain subacount with "-" delimeter

Reply:
fixed.

Reply:
However `data` variable needs to be normalized

Reply:
Ah okay , get it. 
fixed